README.md

```diff
@@ -46,18 +46,18 @@ This is the JavaScript version of [OpenTelemetry](https://opentelemetry.io/), a
 
 ## Compatibility Matrix
 
-| API Version | Core version | Contrib Version         |
-| ----------- |--------------|-------------------------|
-| 0.20.x      | 0.20.x       | ------                  |
-| v1.0.0-rc.0 | 0.19.x       | ------                  |
-| 0.18.x      | 0.18.x       | 0.14.x                  |
-|             | 0.17.x       | ------                  |
-|             | 0.16.x       | ------                  |
-|             | 0.15.x       | 0.13.x                  |
-|             | 0.14.x       | 0.12.x                  |
-|             | 0.13.x       | ------                  |
-|             | 0.12.x       | 0.11.x                  |
-|             | 0.11.x       | 0.10.x                  |
+| API Version                | Core version | Contrib Version         |
+| -------------------------- |--------------|-------------------------|
+| 0.20.x                     | 0.20.x       | 0.20.x                  |
+| ~~v1.0.0-rc.3~~ deprecated | 0.19.x       | 0.16.x                  |
```

Comment:
Why deprecate? 1.0.0-rc.3 + 0.19.x + 0.16.x works as good as e.g. 0.18.x Api + 0.18.x core and 0.14.x contrib.

Reply:
We want people off the 1.x.x-rc.x line as soon as possible. That line of versioning turned out to be a mistake and it causes more confusion for people than good. I'll remove the deprecation warning since as you said it still works.